fwutil/lib.py

```diff
@@ -1050,13 +1036,22 @@ def get_au_status(self):
         if data is None:
             return None
 
-        boot_type = list(data.keys())[0]
-        click.echo("Firmware auto-update performed for {} reboot".format(boot_type))
```

Comment:
Please add these lines to display the boot type for which the update is scheduled.